src/pages/HomePage/styles.scss

```diff
@@ -0,0 +1,48 @@
+$text-color: #000000;
+
+.heading {
+    margin-top: 10px;
+    margin-bottom: 10px;
+    font-weight: bold;
+
+    font-size: 36px;
```

Comment:
this is the case when we can add styles for each header and text into commos file `typography.scss` for example and apply only additional styles.